core/primitives/src/views.rs

```diff
@@ -357,6 +363,17 @@ impl From<BlockHeaderView> for BlockHeader {
     }
 }
 
+#[derive(Serialize, Debug, Clone, BorshDeserialize, BorshSerialize)]
+pub struct BlockHeaderInnerLiteView {
+    pub height: BlockIndex,
+    pub epoch_id: CryptoHash,
+    pub next_epoch_id: CryptoHash,
```

Comment:
Why is epoch id here not `EpochId`?

Reply:
I am not sure why certain decisions are made for views, but `epoch_id` in `BlockHeaderView` is `CryptoHash`, so I kept it consistent.